kubernetes/frontend.yaml

```diff
@@ -12,35 +12,48 @@ spec:
     metadata:
       labels:
         app: frontend
+      annotations:
+        prometheus.io/port: "7980"
+        prometheus.io/scrape: "true"
     spec:
       containers:
       - args:
-        - /config/frontend.json
-        image: buildbarn/bb-storage:20190617T155413Z-3c42fa4
+        - /config/frontend.jsonnet
+        image: buildbarn/bb-storage:20200102T100452Z-f920d92
         name: storage
         ports:
         - containerPort: 8980
           protocol: TCP
+        - containerPort: 7980
```

Comment:
I don't think it's necessary to add containerPort directives for the metrics ports, right? There is, for example, no need to do any service routing to ports like these, so there is no need to make them known to the API server.